src/main/webapp/style.css

```diff
@@ -0,0 +1,150 @@
+body {
+  background-image: url("images/tests.jpg");
+}
+
+.topnav .search-container {
+  float: left;
+}
+
+.topnav input[type=text] {
+  padding: 6px;
+  margin-top: 8px;
+  font-size: 12px;
+  border: none;
+}
+
+.topnav .search-container button {
+  float: right;
+  padding: 6px 10px;
+  margin-top: 8px;
+  background: rgb(211, 206, 209);
+  font-size: 12px;
+  border: none;
+  cursor: pointer;
+}
+
+.topnav .search-container button:hover {
+  background: rgb(202, 24, 24);
+}
+
+.button {
+  background-color: #4CAF50;
+  border: none;
+  color: white;
+  padding: 6px 10px;
+  margin-top: 8px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 12px;
+  margin: 4px 2px;
+  transition-duration: 0.4s;
+  cursor: pointer;
+}
+
+#directions{
+  background-color: rgb(211, 206, 209); 
+  margin-top: 8px;
+  color: black; 
+}
+
+#directions:hover {
+  background-color: rgb(38, 137, 219);
+  color: white;
+}
+
+.sidenav {
+  height: 100%;
+  width: 20%;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background-color: #111;
+  overflow-x: hidden;
+  padding-top: 20px;
+}
+
+.sidenav a {
+  padding: 6px 8px 6px 16px;
+  text-decoration: none;
+  font-size: 25px;
+  color: #818181;
+  display: block;
+}
+
+.sidenav a:hover {
+  color: #f1f1f1;
+}
+
+.main {
+  margin-left: 450px; /* Same as the width of the sidenav */
```

Comment:
With a %width on the left sidebar, and a fixed margin-left on the main ... I suspect this is going to have significant resizing behaviour strangeness...

Reply:
this was a problem i recognised and was thinking of fixing up after approval so rachel can put in the prototype images and button onclicks but I'll see to it rn